src/uct/sm/mm/mm_iface.h

```diff
@@ -92,6 +93,7 @@ struct uct_mm_fifo_element {
 
 struct uct_mm_fifo_ctl {
     volatile uint64_t  head;       /* where to write next */
+    char padding[UCS_SYS_CACHE_LINE_SIZE - 8];
```

Comment:
i think it would be better to use __attribute__((aligned)) here, or at least sizeof(uint64_t) instead of 8